File: benchmark.go
package main

import (
	"fmt"
	"time"
)

func main() {
	current_time := time.Now().UnixNano() / int64(time.Millisecond)
	fmt.Println("********* Bechmark start *********")
	largets_number := 0
	for i := 0; i < 10000000; i++ {
		if largets_number < i {
			largets_number = i
		}
	}
	duration := time.Now().UnixNano()/int64(time.Millisecond) - current_time
	fmt.Println("It takes", duration, "ms\n**********************************\n")

}

// ********* Bechmark start *********
// It takes 7 ms
// **********************************

File: benchmark.rb
current_time = Time.now
puts "********* Bechmark start *********"
largest_number = 0

for x in 1..10000000
  largest_number = x if largest_number < x
end
duration = Time.now - current_time
puts "It takes #{duration*100} ms\n**********************************\n"


# ********* Bechmark start *********
# It takes 56.7782 ms
# **********************************


